FirebaseRemoteConfigSwift/Apps/SwiftUISample/SwiftUISample/ContentView.swift

```diff
@@ -35,10 +36,9 @@ struct ContentView: View {
   @RemoteConfigProperty(
     key: "recipe", fallback: Recipe(recipeName: "banana bread", cookTime: 40, notes: "yum!")
   )
-  var recipe: Recipe
 
-  @State private var realtimeSwitch = false
-  var realtimeToggle: Bool
+  var recipe: Recipe
+  @State private var isChecked = false
```

Comment:
does `isChecked` control something in the UI? looks like the button image only depends on `toggleValue`

Reply:
Good catch! I made it more practical case where isCheck guard the value of the toggle and a toggleStyleSquare config value controls the checkbox style to be square or circle.